tests/cupyx_tests/fallback_mode_tests/test_fallback.py

```diff
@@ -147,3 +147,114 @@ def test_func_not_in_numpy(self):
 
         with self.assertRaises(AttributeError):
             func = fallback_mode.numpy.dummy  # NOQA
+
+
+@testing.gpu
+class TestNotifications(unittest.TestCase):
+
+    def test_1_seterr_geterr(self):
+
+        default = fallback_mode.geterr()
+        assert default == 'warn'
+
+        old = fallback_mode.seterr('ignore')
+        current = fallback_mode.geterr()
+        assert old == 'warn'
+        assert current == 'ignore'
+
+    def test_notification_ignore(self):
+
+        import contextlib
+        from io import StringIO
+
+        saved_stdout = StringIO()
+        with contextlib.redirect_stdout(saved_stdout):
+            fallback_mode.seterr('ignore')
+            res = fallback_mode.numpy.nanargmin([1, 2, 3])  # NOQA
```

Comment:
Don't use `nanargmin` here because `cupy.nanargmin` will be supported in the future.

Reply:
Yes, I was thinking of replacing all tests with a good function. Can you suggest a function other than `vectorize` because it's return type is `numpy.vectorize function`. It can be a function with return type `ndarray`. Is `convolve` or `compress` okay?

Reply:
Tests should not depend on only one function. How about parameterizing with few functions which can take the same arguments?

Reply:
Okay

Reply:
I've added parameterized test, but cannot find enough good functions with same type of arguments. Maybe we can do:
```python
@testing.parameterize(
{'args': (...)}
)
```
and then `func(*args)` something like this, if the tests are not good enough.